packages/wp-source/src/libraries/handlers/__tests__/__snapshots__/date.tests.ts.snap

```diff
@@ -626,7 +638,14 @@ Object {
           "type": "post",
         },
       ],
+      "link": "/2019/?some=param",
       "month": undefined,
```

Comment:
@luisherranz This is not related to the current PR, but `month` and `day` are undefined if the route does not include them, for example the route `/2019/page/2/?some=param` in this test will return `undefined` month and day.

Should we include `day` and `month` conditionally in the final data?

Like we do with `previous` and `next`:

```js
{
  ...(day && { day }),
  ...(month && { month }),
}

Reply:
Yes, good idea!

I'll do the same for `query.s`.